docs/nodes/build/run-avalanche-node-manually.md

```diff
@@ -155,22 +155,22 @@ The resulting folder, `avalanchego-<VERSION>-linux`, contains the binaries.
 If you built from source:
 
 ```sh
-./build/avalanchego
+./build/avalanchego --public-ip=[YOUR NODE'S PUBLIC IP HERE]
```

Comment:
`--public-ip` is not always required, per
https://docs.avax.network/nodes/maintain/avalanchego-config-flags#--public-ip-string

Reply:
Thanks Yulin! Reverted the change, and amended docs/nodes/validate/add-a-validator.md related instructions.